src/index.html

```diff
@@ -1,14 +1,123 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible"
+          content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="home"
+        href="#home"
+        alt="logo"
+      >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item nav_item_"
+          >
```

Comment:
the tag should be written in one line
```suggestion
          <li class="nav_item nav_item_">
```